eng/pipelines/official.yml

```diff
@@ -4,10 +4,9 @@ trigger:
   branches:
     include:
       - dev
-      - release-*-MSRC
+      - release/release-*-MSRC
```

Comment:
Why do we need the `release/` prefix?

The docs here don't really cover that: https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#ci-triggers

Reply:
personally, I don't think this will work. Our release branches are `release-z.y.x`, not `release/release-z.y.x`.

Reply:
@zivkan 
Could you ask from Azdo team about if we need `release/` prefix to make things work? Somehow, I can't sign in and comment under issue you [created](https://developercommunity.visualstudio.com/t/yaml-trigger-starting-builds-on-branches/1681956?space=21&entry=myfeedback).

Reply:
I've been in non-stop meetings all day today, so I haven't had time to yet. As I said in one of those meetings, I am going to.